.github/workflows/lint.yml

```diff
@@ -0,0 +1,73 @@
+name: Lint
+on:
+  workflow_call:
+
+jobs:
+  lint_go:
+    name: Lint Go 🔍
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout code 🛎️
+      uses: actions/checkout@v4
+
+    - name: Set up Go 🏗️
+      uses: actions/setup-go@v5
+      with:
+        go-version-file: go.mod
+        cache: false
+
+    - name: Lint alloy module 🔍
+      uses: golangci/golangci-lint-action@v6
+      with:
+        version: v1.64.7
+
+    - name: Lint syntax module 🔍
```

Comment:
I didn't know we can have emoji 🤣  It's not bad but I wonder if it's really necessary. Most of them are just 🔎  and it might be just extra clutter.

Reply:
😂 There's definitely no practical purpose other than the slight dopamine hit of seeing an emoji here and there. It gets a little more interesting if you use it for things like test 🧪, deploy 🚀, upload 🛫, download 🛬, etc. haha. I often challenge teammates to come up with interesting emojis to describe their workflow steps. Happy to remove them though, since this would be the only workflow containing them and there definitely isn't a ton of variety for this one.